### DISCUSSION TITLE
SetGesture 오류

### DISCUSSION THREAD

**Discussion by hanaaple** [naverz/zepeto-studio-kor#506] [5. Q&A]
버튼을 눌렀을 때 애니메이터 상황인데 어째서 오류인지 애니메이션 클립이 gesture에 들어가지 않네요
애니메이터 State는 gesture인 상태이며 로그로 확인해봤을 때 AnimationClip에 null이 들어가지도 않았습니다
인위적으로 애니메이션 클립을 넣어줄 수 있긴 하겠지만 원래 작동하던 거를 포기하고 싶진 않네요
![image](https://user-images.githubusercontent.com/48403325/159178097-f4046e7c-c127-4120-806f-5796a96db1ab.png)

코드입니다.
![image](https://user-images.githubusercontent.com/48403325/159178442-b2b12e21-9cd3-4354-b183-063b12780933.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 상호작용 버튼 클릭 코드로 보여집니다.
버튼 클릭 btn.OnClick.AddListener() 관련 코드를 Start() 내부로 옮기고,
OnTriggerEnter와 Exit에서는 버튼의 SetActive만 처리하도록 코드 구조를 수정해 보시고 다시 시도해 보시겠어요?

아래 예제 가이드를 따라하시면 제스쳐 항목에 애니메이션 클립이 정상적으로 세팅되오니 아래 내용도 참고 부탁드려요~!
https://github.com/naverz/zepeto-studio-kor/discussions/496#discussioncomment-2362863

**Reply by hanaaple** [reply]
예제 가이드의 스크립트를 복붙하여 실행하였는데도 제스처가 작동하지 않네요...

**Reply by Zepeto-TechEvangelist** [reply]
Hierachy와 Inspector 스크린샷을 올려주시면 문제 해결에 도움이 될 것 같습니다.

**Reply by hanaaple** [reply]
앗 제가 AnimatorOverrideController를 건드리면서 발생한 오류였네요 감사합니다
